docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md

```diff
@@ -66,25 +92,62 @@ Parameter                     | Type                 | Description
 
 If `attrName` or `attrValue` is a String, `withAttribute` selects an element by strict match.
 
+```js
+// Selects div elements whose 'attrName' attribute
+// is set to 'foo'.
+// Does not match the 'attrName1' attribute or the 'foobar' value.
+Selector('div').withAttribute('attrName', 'foo');
```

Comment:
IMO, `or the 'foobar' value.` looks weird in this sentence. 
I'd rephrase it like "Does not match the 'attrName1' attribute or the `attrName` attribute with the 'foobar' value."